extensions/guacamole-auth-sso/modules/guacamole-auth-sso-cas/pom.xml

```diff
@@ -77,9 +77,20 @@
                     <artifactId>commons-codec</artifactId>
                 </exclusion>
 
+                <exclusion>
```

Comment:
Might be useful to document the reason for the exclusions, here?

Reply:
Fixed via rebase.